### PR TITLE
Update permissions

### PR DESCRIPTION
**Pull request by Delphineray**

### CODE REVIEW

File: content/md/permissions-settings/manage-the-interface-and-actions-accesses.md [hunk 1]
```diff
@@ -23,26 +23,40 @@ For almost each screen and button in the PIM, you can say if a user can see/use
...
 | Restore a product _(EE only)_ | User can restore a previous version of a product using the `Restore` button in the product history page  |
 | Manage published products _(EE only)_   | User can access the published products grid by clicking on the `Published products` button  in the `...` dropdown on the top of the product grid |
 
+## Permissions on product models
+
+| Permission name | Effect on the interface |
+|---------------------------|--------------|
+| Consult the categories of a product model  | User can see the `Categories` menu entry in the product form for a product model                                    |
+| Create a product model   | User can create a new product model in the product grid through the `Create` button             |
+| Edit attributes of a product model | User can edit the product’s attributes under the `Attributes` menu entry in the product form for a product model   |
```
Comment (NolwennP) [inline content/md/permissions-settings/manage-the-interface-and-actions-accesses.md]: product model attributes

File: content/md/permissions-settings/manage-the-interface-and-actions-accesses.md [hunk 2]
```diff
@@ -23,26 +23,40 @@ For almost each screen and button in the PIM, you can say if a user can see/use
...
 | Remove an attribute from a product             | User can remove an attribute from a product using the `cross` icon next to attributes added to the product                                                                  |
 | List asset categories in the asset picker _(EE only)_ | User can access the `Manage assets` button to open the asset picker and list the asset categories  |
 | Restore a product _(EE only)_ | User can restore a previous version of a product using the `Restore` button in the product history page  |
 | Manage published products _(EE only)_   | User can access the published products grid by clicking on the `Published products` button  in the `...` dropdown on the top of the product grid |
 
+## Permissions on product models
+
+| Permission name | Effect on the interface |
+|---------------------------|--------------|
+| Consult the categories of a product model  | User can see the `Categories` menu entry in the product form for a product model                                    |
```
Comment (NolwennP) [inline content/md/permissions-settings/manage-the-interface-and-actions-accesses.md]: Should we call the product form, a `product model form`?

File: content/md/permissions-settings/manage-the-interface-and-actions-accesses.md [hunk 3]
```diff
@@ -23,26 +23,40 @@ For almost each screen and button in the PIM, you can say if a user can see/use
...
+| Create a product model   | User can create a new product model in the product grid through the `Create` button             |
+| Edit attributes of a product model | User can edit the product’s attributes under the `Attributes` menu entry in the product form for a product model   |
+| View product model history | User can see the `History` menu entry in the product form for a product model  |
+| Remove a product model  | User can remove a product model from the PIM using the `Delete` button in the `...` dropdown on the top of the product form or the product grid (trash icon)     |
```
Comment (NolwennP) [inline content/md/permissions-settings/manage-the-interface-and-actions-accesses.md]: `or using directly the trash icon in the product grid`

File: content/md/permissions-settings/manage-the-interface-and-actions-accesses.md [hunk 4]
```diff
@@ -23,26 +23,40 @@ For almost each screen and button in the PIM, you can say if a user can see/use
...
 | Download the product as PDF                | User will be able to generate a PDF file containing all the product information using the `PDF` button in the `...` dropdown on the top of the product form |
 | Edit attributes of a product                   | User can edit the product’s attributes under the `Attributes` menu entry of the product form   |
 | View product history | User can see the `History` menu entry in the product form  |
 | List products                                  | User can see the list of products in several pages: on the product grid in `Products`, as well as on the `Associations` menu entry of the product form, in `Groups`... |
 | Product mass edit actions | User can make mass actions on the product grid |
-| Remove a product           | User can remove a product from the PIM using the `Delete` button in the `...` dropdown on the top of the product form      |
+| Remove a product           | User can remove a product from the PIM using the `Delete` button in the `...` dropdown on the top of the product form or the product grid (trash icon)      |
```
Comment (NolwennP) [inline content/md/permissions-settings/manage-the-interface-and-actions-accesses.md]: `or using directly the trash icon in the product grid`

File: content/md/permissions-settings/manage-the-interface-and-actions-accesses.md (hunk 2 above)
Comment (Delphineray) [reply content/md/permissions-settings/manage-the-interface-and-actions-accesses.md]: nope ;)

File: content/md/permissions-settings/manage-the-interface-and-actions-accesses.md (hunk 1 above)
Comment (Delphineray) [reply content/md/permissions-settings/manage-the-interface-and-actions-accesses.md]: done

File: content/md/permissions-settings/manage-the-interface-and-actions-accesses.md (hunk 3 above)
Comment (Delphineray) [reply content/md/permissions-settings/manage-the-interface-and-actions-accesses.md]: ok

File: content/md/permissions-settings/manage-the-interface-and-actions-accesses.md (hunk 4 above)
Comment (Delphineray) [reply content/md/permissions-settings/manage-the-interface-and-actions-accesses.md]: done

**Pull request merged**